Controllers/Backend/SwagBackendOrder.php

```diff
@@ -221,7 +221,9 @@ public function getPaymentAction(): void
         $builder = $this->get('models')->createQueryBuilder();
         $builder->select(['payment'])
             ->from(Payment::class, 'payment')
-            ->orderBy('payment.active', 'DESC');
+            ->where('payment.active = :isActive')
+            ->setParameter('isActive', 1)
+            ->orderBy('payment.description', 'ASC');
```

Comment:
the shipping methods are sorted by position. I guess we should make it consistent and do the same with the payment methods